src/main/java/utask/commons/util/UpdateUtil.java

```diff
@@ -78,23 +82,22 @@ public static Task createEditedTask(ReadOnlyTask taskToEdit, Boolean value)
         case FLOATING:
             placeholder = new FloatingTask(taskToEdit.getName(),
```

Comment:
If you often create different instances like this.  Consider group these factory-like methods to a helper to generate instances.